viewer/packages/camera-manager/src/Flexible/FlexibleCameraMarkers.ts

```diff
@@ -75,9 +91,13 @@ function setPosition(object3D: Object3D, position: Vector3, manager: FlexibleCam
   object3D.updateMatrixWorld();
 }
 
-function createSprite(outerColor: string, innerColor: string): Sprite {
-  const texture = createTexture(25, 3, outerColor, innerColor);
-  const material = new SpriteMaterial({ map: texture, depthTest: false });
+function createSprite(options: FlexibleControlsOptions, depthTest: boolean): Sprite {
+  const texture = createTexture(25, 3, options.outerMarkerColor, options.innerMarkerColor);
```

Comment:
I would suggest to move these magic numbers to a class constants instead

Reply:
Yes, I move it to constants like this:

```
const TEXTURE_SIZE = 25;
const LINE_WIDTH = 3;
const NO_DEPTH_TEST_OPACITY = 0.5;

```